qt/translations/transmission_sl.ts

```diff
@@ -1815,14 +1815,14 @@ Za dodajanje nadaljnjega glavnega URL, ga dodajte za prazno vrstico.</translatio
         <translation type="unfinished"></translation>
     </message>
     <message numerus="yes">
-        <location line="+19"/>
-        <source> minute(s) ago</source>
+        <location filename="PrefsDialog.cc" line="582"/>
```

Comment:
Please revert changes to this file. Translations are done through Transifex and synced down from there every now and then. I'll make sure the updated phrase is available there once this PR is merged.